src/org/opendatakit/briefcase/ui/reused/FocusAdapterBuilder.java

```diff
@@ -0,0 +1,36 @@
+package org.opendatakit.briefcase.ui.reused;
+
+import java.awt.event.FocusAdapter;
```

Comment:
The Swing and AWT adapters are great when you don’t want to implement all the interface’s methods, but here you do, so the adapter is an unnecessary layer, n'est-ce pas? Also notice that IDEA can now point out that onFocusGained is never called (for whatever that’s worth—I assume you included it for completeness).

<img width="1618" alt="Screen Shot 2019-07-28 at 12 34 21 PM" src="https://user-images.githubusercontent.com/55099/62011958-76bca100-b134-11e9-9c81-0180f54e1242.png">